samples/CommunityToolkit.Maui.Sample/Pages/Views/AvatarView/AvatarViewGesturesPage.xaml.cs

```diff
@@ -0,0 +1,19 @@
+using System.Diagnostics;
+using CommunityToolkit.Maui.Sample.ViewModels.Views.AvatarView;
+
+namespace CommunityToolkit.Maui.Sample.Pages.Views;
+
+public partial class AvatarViewGesturesPage : BasePage<AvatarViewGesturesViewModel>
+{
+	public AvatarViewGesturesPage(AvatarViewGesturesViewModel avatarViewGesturesViewModel) : base(avatarViewGesturesViewModel) => InitializeComponent();
+
+	void DragGestureRecognizer_DragStarting(object sender, DragStartingEventArgs e) => Debug.WriteLine("AvatarView drag gesture recognizer, drag starting.");
```

Comment:
Let's use Toast to display the result.

Debug WriteLine is not visible on UI and user may assume the app doesn't work

Reply:
Great idea!
I have changed to use as short Toast instead of debug.